api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -531,6 +531,11 @@
      */
     Attribute<String> LIBRARY_JARS = new Attribute<>(String2String.getInstance());
 
+    /**
+     * Configuration file for custom SSL keystore to override ssl-server.xml
+     */
+    Attribute<String> CUSTOM_SSL_SERVER_CONFIG = new Attribute<>(String2String.getInstance());
```

Comment:
There isn't anything specific to ssl here isn't it. It is custom config that is being passed to stram webservice for initialization. There is a property for stram web service authentication (STRAM_HTTP_AUTHENTICATION). I suggest this property follow the same format, maybe STRAM_HTTP_CUSTOM_CONFIG? I understand it is a little bit of a misnomer because this is primarily for https but I think it is better to be consistent for the user. In the javadoc you can include information such as this config can be used to configure ssl parameters and also that this file needs to be present on all the nodes.